xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/job/question/XClassBreakingQuestion.data.vm

```diff
@@ -124,9 +124,6 @@ $response.setContentType("application/json")
       'children': true,
       'state': {
         'selected': $selected
-      },
-      'data': {
-        'link': $pageDoc.getURL('view')
```

Comment:
Having a link on the tree node is still useful even if not by clicking on it but by opening the link on a new browser tab. I would do it using the [``a_attr``](https://www.jstree.com/docs/json/) property though.

Reply:
I think the problem would be the same since the link is around the checkbox too: so selecting the element would always open the link, which is not expected by the user...

Reply:
The example from https://www.jstree.com/plugins/ (look for the checkbox plugin) has nodes with links. The browser shows the URL when you hover the node and you have the option to open the link in a new browser tab when opening the contextual menu (Right click).

Also, I remember the [tree widget](https://extensions.xwiki.org/xwiki/bin/view/Extension/Tree%20Widget) has a special ``jstree-no-links`` CSS class.

Reply:
So a link with `jstree-no-links` would only work when right clicking on it?

Reply:
That's the behavior of the location picker tree from the Create Page form. Clicking on a node selects the node but doesn't follow the link, and you can open the link in a new browser tab. The checkbox plugin is not used there though.

Reply:
Hm ok. I'll check if you think it can be helpful, but frankly as the link is a bit "hidden" because of the checkbox, I'm not sure it worths it to keep it here.
I assume that in most cases, the user would open a new tab a go manually to the appropriate page if needed.